README.md

```diff
@@ -37,6 +37,31 @@ $ docker run --detach \
              thelounge/thelounge:latest
 ```
 
+### Executing commands in the container
+
+Due to the way root permissions are dropped in the container, it's highly recommended to pass the `--user node` argument to any
+commands you execute in the container via Docker to ensure that file permissions retain the correct owner, like so:
+
+```
+$ docker exec --user node -it [container_name] thelounge add MyUser
+```
+
+### Configuring identd
+
+Since root permissions are dropped in the container the default port 113 can not be used as it is within the
```

Comment:
Can you throw a link to https://thelounge.chat/docs/guides/identd-and-oidentd somewhere in here and use 9001 as the example port instead of 1113 to match the guide.